api/v1alpha1/repo_manager_types.go

```diff
@@ -262,6 +262,11 @@ type PulpSpec struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
 	Cache Cache `json:"cache,omitempty"`
 
+	// Galaxy defines the resources that are specific to Galaxy deployments
+	// +kubebuilder:validation:Optional
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:hidden"}
+	Galaxy Galaxy `json:"galaxy,omitempty"`
```

Comment:
I'm creating a galaxy field because:
* we can use it if more configurations specific to galaxy deployments appear
* Pulp users who don't know about Galaxy will not be confused with galaxy configs, for example, a Pulp user could get confused on filling `.spec.ee_defaults` field, but defining it as `.spec.galaxy.ee_defaults` user will get a better idea that this is something for galaxy

WDYT?